.github/actions/deploy-backend/action.yml

```diff
@@ -152,7 +151,7 @@ runs:
       shell: bash
       env:
         SSL_KEY: ${{ steps.key-vault.outputs[env.KEY_NAME] }}
-    
+
```

Comment:
The system is randomly adding new lines and I guess it's related to Windows/Linux \n.
This should not be an issue.

Reply:
I've set my editor to adde a newline and remove trailing spaces on save.  This is more for consistency than anything else.